indico/modules/events/abstracts/views.py

```diff
@@ -78,6 +78,14 @@ class WPDisplayAbstracts(WPDisplayAbstractsBase):
     menu_entry_name = 'call_for_abstracts'
 
 
+class WPDisplayCallForAbstracts(WPDisplayAbstracts):
+    template_prefix = WPDisplayAbstracts.template_prefix + 'display/'
+
+    def getJSFiles(self):
+        return (WPDisplayAbstractsBase.getJSFiles(self) +
+                self._asset_env['modules_event_display_js'].urls())
```

Comment:
![does](https://cloud.githubusercontent.com/assets/9071624/20564943/ba0e036a-b18f-11e6-863a-71da7090a324.png)
it does but its like this in all other classes as well so i thought its some kind of a pattern